Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -42,41 +42,72 @@ const addPlatformObject = runtimeScene => {
   return platform;
 };
 
-describe('gdjs.PlatformerObjectRuntimeBehavior', function() {
-  const runtimeScene = makeTestRuntimeScene();
-
-  // Put a platformer object in the air.
-  var object = new gdjs.RuntimeObject(runtimeScene, {
-    name: 'obj1',
+const addLadderObject = runtimeScene => {
```

Comment:
Nitpicking: let's put these two functions inside the `describe('gdjs.PlatformerObjectRuntimeBehavior', function() { `

So that everything is contained and no risk of leaking in the global namespace :) 
(I should have done this already!)